nautobot/dcim/models/locations.py

```diff
@@ -251,6 +269,32 @@ class Meta:
     def __str__(self):
         return self.name
 
+    @classmethod
+    def get_natural_key_fields(cls):
+        natural_key = []
+        name = "name"
+        for _ in range(cls.objects.max_tree_depth() + 1):
```

Comment:
Nevermind, I see now. This is why you're explicitly returning `0` from `max_tree_depth`